src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/handlers/presubmit/CustomiseHearingBundleHandler.java

```diff
@@ -305,10 +307,7 @@ private void restoreCollections(
         });
     }
 
-    private void restoreReheardDocumentsInCollections(AsylumCase asylumCase, AsylumCase asylumCaseBefore, boolean isRemittedFeature, AsylumCaseDefinition latestField, AsylumCaseDefinition existingField) {
-        if (!isRemittedFeature) {
```

Comment:
Why did you deleted this remitted check? I thought that this should be done only for remitted.

Reply:
No, the way of storing of documents was changed as a part of set-aside release, and the requirement was to add remitted documents in the bundle if it is remitted. There is another scenario where the FTPA is decided as rule 35 and the remitted path is skipped. To cater that situation, we don't need this remitted check for "Reheard hearing documents" & "Decision & Reason documents".